snap-src/bin/config

```diff
@@ -108,7 +108,7 @@ DESCRIPTION_BIGEVENTS_PATTERN="Big events pattern: Notify always due etc regardl
 DEFAULT_BIGEVENTS_PATTERN="NONE"
 KEY_BIGEVENTS_PATTERN="bigevents-pattern"
 
-DESCRIPTION_NOTIFY_DUE_DAYS_BEFORE_AND_AFTER="Notify due days, default 2 days before and after. 0 = due notifications disabled. Default: 2"
+DESCRIPTION_NOTIFY_DUE_DAYS_BEFORE_AND_AFTER="Notify due days, accepts ',' delimited string, i.e. 2,0 means notify will be sent out 2 days before and right on due day. Default: empty"
```

Comment:
Please add this change also to:
- wekan/snap-src/wekan-help
- wekan/docker-compose.yml
- wekan/torodb-postgresql/docker-compose.yml
- wekan/start-wekan.sh
- wekan/start-wekan.bat
- wekan/releases/virtualbox/start-wekan.sh